app/src/main/res/layout/activity_movie_details.xml

```diff
@@ -0,0 +1,242 @@
+<?xml version="1.0" encoding="utf-8"?>
+<ScrollView xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:background="@drawable/movie_details_bg">
```

Comment:
Figma doesn't provide a background picture for landscape mode. Maybe there is a way to make vertical image look good in landscape mode, though I couldn't find